subworkflows/local/bgc.nf

```diff
@@ -148,6 +157,15 @@ workflow BGC {
     }
 
     // COMBGC
+
+    ch_bgcresults_for_combgc
+        .join(fastas, remainder: true)
+        .filter {
+            meta, bgcfile, fasta ->
+                if ( !bgcfile ) { log.warn("[nf-core/funcscan] BGC workflow: No hits found by BGC tools; comBGC summary tool will not be run for sample: ${meta.id}") }
+                return [meta, bgcfile, fasta]
+        }
```

Comment:
Jsutto remind me, this is purely for reporting correct? If so, why not just add the filter bit to after (new) line 87, given you are joining `fastas` there?

Although if this works it works, it's suboptimal but can clean this up anotehr time

Reply:
In line 87 it would be only within the antismash block. We don't want to report per tool, that would be an overkill of warning messages. So I'd rather keep the warning here to check if _none of all_ tools found hits for a sample.

Reply:
Got it thanks!